site2/docs/admin-api-topics.md

```diff
@@ -1138,12 +1138,18 @@ $ pulsar-admin topics get-partitioned-topic-metadata \
 ```
 
 <!--REST API-->
```

Comment:
Here we have `<!--REST API-->`  and `<!--Java-->` methods, do we have `<!--pulsar-admin-->` method?

Reply:
No,  pulsar-admin does not have this method